_includes/corevocabulary.md

```diff
@@ -3,6 +3,10 @@
 Despite its flexible and extensible nature, Hyper is a very simple
 media type with a small vocabulary.
 
+In Hyper, JSON arrays are used as sets, not ordered lists.  The
+order of a specified array should not be considered part of the
+conveyed information.
```

Comment:
This is highly problematic. Hyper is a set of mixins to JSON. In JSON arrays are ordered lists. Saying that Hyper uses JSON arrays as sets alters semantics of arrays in JSON and that is a problem. Also it is unnecessary. Just because our core mixins don't care about the order of items in JSON arrays, doesn't mean that we assert the complete absence of such meaningful ordering. The data part of the Hyper message may contain some arrays where ordering is important - we don't prohibit it and we shouldn't get into it.

Reply:
Yes, that makes sense.   I still think we should clarify that arrays in different orders should not be considered different for `rel`, etc.

Reply:
Yeah. That's fair. We probably should do it case-by-case, even if it is more work. Sorry!